tree.go

```diff
@@ -794,7 +794,20 @@ func (n *InternalNode) GetProofItems(keys keylist) (*ProofElements, []byte, [][]
 	for i, child := range n.children {
 		fiPtrs[i] = &fi[i]
 		if child != nil {
-			points[i] = child.Commitment()
+			var c VerkleNode
+			if _, ok := child.(HashedNode); ok {
```

Comment:
Maybe add a comment TODO to remember to remove this when `BatchSerialize` doesn't hash nodes.